docs/04-create-addons.md

```diff
@@ -0,0 +1,116 @@
+
+|  Field Name | Required |      Description               |
+|-----------|:--------:|------------------------------------|
+|     **name**    |   Yes   |     The name of the plan.   |
+|      **id**     |   Yes   |     The ID of the plan. |
+| **description** |   Yes   | The description of the plan. |
+| **displayName** |   Yes   | The display name of the plan. |
+|  **bindable**   |   No  | The field that specifies whether you can bind an instance of the plan or not. The default value is `false`. |
+|     **free**    |   No  | The attribute which specifies whether an instance of the plan is free or not. The default value is `false`.    |
+
+* `bind.yaml` file - contains information about binding in a specific plan. If you define in the `meta.yaml` file that your plan is bindable, you must also create a `bind.yaml` file. For more information, read about [binding addons](https://github.com/kyma-project/helm-broker/blob/master/docs/05-bind-addons.md).
+
+* `values.yaml` file - provides the default configuration values in a given plan for the chart definition located in the `chart` directory. For more information, see the [values files](https://github.com/kubernetes/helm/blob/release-2.6/docs/chart_template_guide/values_files.md) specification.
+
+* `create-instance-schema.json` file - contains a schema that defines parameters for a provision operation of a ServiceInstance. Each input parameter is expressed as a property within a JSON object.
+
+* `update-instance-schema.json` file - contains a schema that defines parameters for an update operation of a ServiceInstance. Each input parameter is expressed as a property within a JSON object.
+
+* `bind-instance-schema.json` file - contains a schema that defines parameters for a bind operation. Each input parameter is expressed as a property within a JSON object.
+
+>**NOTE:** For more information about schemas, see the [specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#schemas-object).
+
+## docs directory
+
+In the `docs` directory, provide documentation for your addon. The documentation can include Markdown documents, AsyncAPI, OData, and OpenAPI specification files. Create the `assets` directory inside the `docs` directory to store assets, such as images. The `docs` directory must contain a `meta.yaml` file, which provides information on how documentation for the addon is uploaded.
+Because you can install the Helm Broker as a ClusterServiceBroker or as a ServiceBroker, documentation for addons is provided using either [ClusterAssetGroups](https://kyma-project.io/docs/components/rafter#custom-resource-cluster-asset-group) or [AssetGroups](https://kyma-project.io/docs/components/rafter#custom-resource-asset-group) custom resources, respectively.
```

Comment:
```suggestion
Because you can install Helm Broker as a ClusterServiceBroker or as a ServiceBroker, documentation for addons is provided using either [ClusterAssetGroups](https://kyma-project.io/docs/components/rafter#custom-resource-cluster-asset-group) or [AssetGroups](https://kyma-project.io/docs/components/rafter#custom-resource-asset-group) custom resources, respectively.
```